pages/common/git-extras.md

```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Git extension pack.
+> More information: <https://github.com/tj/git-extras>.
+
+- Download and/or upgrade `git-extras` commands:
```

Comment:
I read the thread about this example description, but why just `download`? why not?
```suggestion
- Install or upgrade `git-extras` commands:
```
`Download` sounds like `get a tarball file in the current directory` :man_shrugging:

Reply:
Sure.

Reply:
Sounds ok to you @bl-ue?